### ISSUE TITLE
JS errors in documentation examples

### ISSUE THREAD

**Issue by AucT** [twbs/bootstrap#20218]
just run any example from docs like http://v4-alpha.getbootstrap.com/examples/navbar-top-fixed/ and open console
```
Uncaught Error: Bootstrap tooltips require Tether (http://github.hubspot.com/tether/)
```

**Comment by antmatr** [comment]
Correct me if I'm wrong, but it seems like on [actual site](http://getbootstrap.com/) version everything is fine with [this example](http://getbootstrap.com/examples/navbar-fixed-top/) and with [others](http://getbootstrap.com/examples/) too. Examples from repository also do not throw any errors.

**Comment by tillkruss** [comment]
@antmatr: Only v4 is broken: http://v4-alpha.getbootstrap.com/examples/navbar-top-fixed/

**Comment by wolfy1339** [comment]
You can safely ignore that error for any pages that aren't using popovers or tooltips

**Comment by AucT** [comment]
@wolfy1339
Not when you use gulp to mix all in 1 file

**Comment by bardiharborow** [comment]
This is fixed by 95d030c.

**Comment by mdo** [comment]
Closing per previous comment then.
